com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -564,8 +584,16 @@ private void NetworkVariableUpdate(ulong clientId, int behaviourIndex)
 
                             if (writtenAny)
                             {
-                                NetworkManager.MessageSender.Send(clientId, NetworkConstants.NETWORK_VARIABLE_DELTA,
-                                    m_ChannelsForNetworkVariableGroups[j], buffer);
+                                var context = NetworkManager.MessageQueueContainer.EnterInternalCommandContext(
```

Comment:
Cool design.  Did you consider passing the body (e.g. `nonNullContext.NetworkWriter.WriteBytes(buffer.GetBuffer(), buffer.Length);` in as a lambda?  Then the `EnterInternalCommandContext` can do the checking / set up the nonNullContext?

Reply:
I considered it, but passed on that design because creating the lambda would result in an allocation, and I didn't want to add that as an unavoidable requirement for the interface.

Also, I'm adjusting the way this works in my serialization RFC, and that'll eliminate the need for this boilerplate, so it's just temporary for now. 😄